source/elements/oneMKL/source/domains/dft/enums_and_config_params.rst

```diff
@@ -152,29 +153,31 @@ config_param
       * -   :ref:`NUMBER_OF_TRANSFORMS<onemkl_dft_config_number_of_transforms>`
         -   If you need to perform a large number of identical DFTs, you can do this in a single call to a compute_forward function with the value of this equal to the actual number of the transforms. Takes a value of ``std::int64_t`` with default value of 1.
       * -   :ref:`onemkl_dft_complex_storage`
-        -   Specifies the data storage format for :ref:`onemkl_dft_enum_domain` with value of ``COMPLEX``. 
-      * -   :ref:`onemkl_dft_real_storage`
-        -   Specifies the data storage format for :ref:`onemkl_dft_enum_domain` with value of ``REAL``.
-      * -   :ref:`onemkl_dft_conjugate_even_storage`
-        -   Specifies the data storage format using conjugate-even symmetry of the data which allows to store only half of the mathematical results.
+        -   Specifies the forward- and backward-domain data storage format for descriptors of ``COMPLEX`` template :ref:`onemkl_dft_enum_domain`.
+      * -   :ref:`REAL_STORAGE<onemkl_dft_real_and_conjugate_even_storage>`
```

Comment:
I had the same thoughts in my first attempts but that would have triggered a lot more changes so I must confess that I had chosen this weird mid-point solution, hoping that no-one would point that out as that was referred to elsewhere too (in pages for compute functions) and addressing it would have required a full revision of it all :-D 
Anyhow, this is what I have done, I believe ``REAL_STORAGE`` and ``CONJUGATE_EVEN_STORAGE`` have now disappeared from the specs. Please let me know if you are fine with this form with a thumbs-up or comment.